go/iden3mobile/eventmanager.go

```diff
@@ -15,144 +13,66 @@ type eventToStore struct {
 	Err string
 }
 
+type Sender interface {
+	Send(*Event)
+}
 type EventManager struct {
-	m            sync.Mutex
 	storage      db.Storage
-	eventQueue   chan Event
-	next         sync.WaitGroup
 	unreadEvents *db.StorageList
```

Comment:
Maybe now it shouldn't be called unreadEvents since they are just "stored events"?